test/e2e/serial/tests/workload_placement.go

```diff
@@ -746,7 +746,7 @@ func maxResourceType(nrtInfo nrtv1alpha1.NodeResourceTopology, resName corev1.Re
 			continue
 		}
 
-		if zoneQty.Cmp(max) > 1 {
+		if zoneQty.Cmp(max) == 1 {
```

Comment:
why?

Reply:
because resource.cmp return -1, 0, 1 when zoneQty is less, equal, greater than max. having it > 1 is always false thus max will always be 0. 
resource:
https://github.com/openshift-kni/numaresources-operator/blob/3289fb4fdb391b018fd98ec2fe2c5f95030d7c63/vendor/k8s.io/apimachinery/pkg/api/resource/quantity.go#L590
did I miss something?

Reply:
Nope, actually now I miss how this could have possibly worked before :) good catch!